components/apimgt/org.wso2.carbon.apimgt.impl/src/test/java/org/wso2/carbon/apimgt/impl/workflow/UserSignUpSimpleWorkflowExecutorTest.java

```diff
@@ -103,20 +100,16 @@ public void testRetrievingWorkflowType() {
 
     @Test
     public void testExecutingUserSignUpSimpleWorkflow() throws APIManagementException, UserStoreException {
-        Map<String, Boolean> roleMap = new HashMap<String, Boolean>();
-        roleMap.put(signUpRole, false);
+        ArrayList<String> roleMap = new ArrayList<String>();
```

Comment:
Let's make this as:
```java
List<String> roleMap = new ArrayList<String>();
```

There are a few other similar instances in this PR. Let's check them too.

Reply:
Fixed in https://github.com/wso2/carbon-apimgt/pull/11951